Dockerfile

```diff
@@ -1,17 +1,45 @@
-FROM node:16.14-alpine3.14
+ARG IMG=node:16.14-alpine3.14
+
+FROM $IMG as builder
+
 WORKDIR /usr/src/app/
 
 COPY .babelrc package.json package-lock.json ./
 RUN apk add --no-cache --virtual .gyp python3 make g++ &&\
-    apk add dumb-init &&\
-    npm ci --only=production &&\
+    npm install &&\
```

Comment:
I understand why we're using the `install` command instead of `ci`, but it's unusual to see this in a production environment.

What do you think about adding a comment here just expliciting that the `install` is needed for the `builder` environment?

Reply:
Done